src/main/openapi/ckan.yaml

```diff
@@ -166,6 +166,10 @@ components:
           type: array
           items:
             type: string
+        keywords:
```

Comment:
@brunopacheco1  is this correct name? Still looking in ckan documentation to find it

Reply:
Unfortunetally no, it's `tags`.

Reply:
And it is a complex object:

```yaml
CkanTag:
      type: object
      properties:
        id:
            type: string
        state:
           type: stirng
        display_name:
          type: string
        name:
          type: string      
      required:
        - id
        - display_name
        - name
        - state
```